data/explorations/test_interactions/Test of expressions and interactions.yaml

```diff
@@ -1,578 +1,693 @@
-author_notes: ''
-auto_tts_enabled: true
-blurb: ''
-category: Test
-correctness_feedback_enabled: false
 init_state_name: Continue
```

Comment:
Why is this file changed so much -- could you make the diff smaller?

In fact, for all yaml files in this PR, please make sure the keys are sorted alphabetically.

Reply:
done, sorted the keys